tests/test_validation.py

```diff
@@ -84,19 +84,19 @@ def spec_1bin_lumi(source=source_1bin_example1()):
                 "name": "channel1",
                 "samples": [
                     {
-                        "data": [20.0, 10.0],
+                        "data": [20.0],
```

Comment:
As this has been migrated out into PR #564 this could get dropped if you want to get this in ASAP. If you don't mind waiting a bit, it will get naturally cleaned up in the rebase.